build/import/VisualStudio.props

```diff
@@ -2,6 +2,11 @@
 <Project>
   <Import Project="HostAgnostic.props"/>
 
+  <ItemGroup>
```

Comment:
Please copy the existing pattern:

1) Add these to the other PackageReferences below:
2) Add a PackageReference without a verisons
3) Put an Update with the version in Packages.props.

Reply:
when you say Packages.props, do you wanted to say Packages.targets ?